scripts/base/frameworks/cluster/main.zeek

```diff
@@ -363,7 +363,9 @@ export {
 	## WebSocket server options to pass to :zeek:see:`Cluster::listen_websocket`.
 	type WebSocketServerOptions: record {
 		## The host address to listen on.
-		listen_host: string;
+		listen_host: string &optional &deprecated="Use $listen_addr instead.";
```

Comment:
@ckreibich - using `addr` here for listening makes sense to me and Benjamin. There is some confusion with #2729 and the existing Broker API using `string` as a `a` parameter. Was #2729 ever meant for listening on addresses, or was that more about connection establishment should be able to use names rather than addresses?